pkgs/tools/system/netdata/default.nix

```diff
@@ -10,14 +20,20 @@ stdenv.mkDerivation rec{
   };
 
   nativeBuildInputs = [ autoreconfHook pkgconfig ];
-  buildInputs = [ zlib ]
-    ++ (if stdenv.isDarwin then [ libossp_uuid CoreFoundation IOKit ] else [ libuuid ]);
+  buildInputs = [ curl.dev libcap.dev zlib.dev ]
```

Comment:
`libcap` should be only included on non-macOS: https://logs.nix.ci/?key=nixos/nixpkgs.64241&attempt_id=bff9eca1-8826-4b1d-9d18-1a3c80d2a2be